modules/apps/portal-search/portal-search-ranking-web/src/main/resources/META-INF/resources/js/components/ResultsRankingForm.es.js

```diff
@@ -28,6 +28,7 @@ class ResultsRankingForm extends Component {
 		cancelUrl: PropTypes.string.isRequired,
 		fetchDocumentsHiddenUrl: PropTypes.string.isRequired,
 		fetchDocumentsUrl: PropTypes.string.isRequired,
+		initialAliases: PropTypes.arrayOf(String).isRequired,
```

Comment:
Let's not require aliases and default it to an empty array since it shouldn't be mandatory that the server returns an empty array.